radio/src/targets/horus/telemetry_driver.cpp

```diff
@@ -42,6 +47,11 @@ static void telemetryInitDirPin()
 
 void telemetryPortInit(uint32_t baudrate, uint8_t mode)
 {
+//OW
+#if defined(TELEMETRY_MAVLINK)
+  if (g_eeGeneral.mavlinkExternal == 1) return;
```

Comment:
Why is it necessary to disable telemetry in general the hard way? This will break use-cases.